packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -71,7 +76,8 @@ export function ListViewItem(props) {
       );
   }
 
-  let showCheckbox = state.selectionManager.selectionMode !== 'none';
+  let showCheckbox = state.selectionManager.selectionMode !== 'none' && state.selectionManager.selectionBehavior === 'toggle';
+  let isSelected = state.selectionManager.isSelected(item.key);
```

Comment:
Something broke with the merge from master, looks like the selectedKey isn't being triggered on row press, digging into why